### DISCUSSION TITLE
`next start` cannot access a new file which downloads while runtime

### DISCUSSION THREAD

**Discussion by tirogen** [vercel/next.js#15441] [Help] [answered]
My problem seem [https://github.com/vercel/next.js/issues/10256](https://github.com/vercel/next.js/issues/10256)

Inside `getServerSideProps` of `somepage.js` has a part of code which is download a image and push it into public/images directory.

If I command `next dev`, then go to `https://localhost:3000/somepage`, a picture will be downloaded and it can access from `http://localhost:3000/images/cat.jpg`

but, If I command `next start`, then go to `https://localhost:3000/somepage`, a picture will be downloaded, but the page `http://localhost:3000/images/cat.jpg` show `404 This page could not be found.`

Can anyone help me how to make the image that can accessable if I command `next start`

**Comment by jamesmosier** [comment, marked as answer]
Hi @tirogen. There are a few other discussions which touch on this subject, you may find them helpful:

* https://github.com/vercel/next.js/discussions/15089#discussioncomment-36357
* https://github.com/vercel/next.js/discussions/13351#discussioncomment-18290
* https://github.com/vercel/next.js/discussions/14769#discussioncomment-32594

with that said, this will not work. The `public` folder is not meant for dynamic files after the build occurs. You'll want to upload any images to somewhere like S3 or Google Cloud.
